web/js/modules/screen-size/reducer.js

```diff
@@ -0,0 +1,31 @@
+import { isMobileOnly, isTablet } from 'react-device-detect';
+import { SET_SCREEN_INFO } from './constants';
+
+export const initialState = {
+  screenHeight: '',
+  screenWidth: '',
+  isMobileDevice: false,
+  orientation: '',
+};
+
+export const getInitialState = () => ({
```

Comment:
Not sure that we need to do all this here in `getInitialState` since the same code is run when the app initializes in `app.js`

Reply:
Removed most variables from get initial state.